comms/core/src/peer_manager/storage/database.rs

```diff
@@ -1074,24 +1098,27 @@ impl PeerDatabaseSql {
         let mut results;
         if start_bytes.len() > NodeId::byte_size() {
             results = peers::table
-                .inner_join(multi_addresses::table.on(multi_addresses::peer_id.eq(peers::peer_id)))
+                .left_outer_join(multi_addresses::table.on(multi_addresses::peer_id.eq(peers::peer_id)))
```

Comment:
these should be left_join
Not outer join. 
outer join returns all matches in both tables.